scripts/application_to_provider.py

```diff
@@ -581,7 +581,7 @@ def migrate_image_data_glance(src_glance_client, dst_glance_client, img_uuid, lo
         with open(local_path, 'rb') as img_file:
             try:
                 # "Upload" method is different for Glance client v1, than for v2
-                if type(dst_glance_client) is glanceclient.v1.client.Client:
+                if hasattr(glanceclient, 'v1') and type(dst_glance_client) is glanceclient.v1.client.Client:
```

Comment:
Some thoughts.

We call an rtwo method to construct the client for us, how it does that isn't our concern. The package structure of the glance library that rtwo uses has changed, but that shouldn't matter here. This script already gets passed a glance version `dst_glance_client_version`, so it has its own representation of that state, inspecting the class for a version is a redundant representation of that state.

This file actually imports glanceclient, so this script is using an interface that its library no longer supports.

This method above should just be parametrized with the version or the path to the class should be corrected to match its library.